src/main/java/com/agency04/sbss/pizza/rest/CustomerRestController.java

```diff
@@ -1,32 +1,29 @@
 package com.agency04.sbss.pizza.rest;
 
+import com.agency04.sbss.pizza.DAO.CustomerRepository;
 import com.agency04.sbss.pizza.model.Customer;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import javax.annotation.PostConstruct;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
+import java.util.Optional;
+
 
 @RestController
 @RequestMapping("/api/customer")
 public class CustomerRestController {
 
-    private List<Customer> customers;
+    private CustomerRepository customerRepository;
```

Comment:
Zašto se u Controller klasi direktno pristupa repository sloju? Potrebno je dodati servisni sloj koji će pristupati repository sloju. Ovo vrijedi za sve Controller klase.